src/csharp/Grpc.Core.Api/Interceptors/InterceptingCallInvoker.cs

```diff
@@ -43,7 +43,7 @@ public InterceptingCallInvoker(CallInvoker invoker, Interceptor interceptor)
         /// <summary>
         /// Intercepts a simple blocking call with the registered interceptor.
         /// </summary>
-        public override TResponse BlockingUnaryCall<TRequest, TResponse>(Method<TRequest, TResponse> method, string host, CallOptions options, TRequest request)
+        public override TResponse BlockingUnaryCall<TRequest, TResponse>(Method<TRequest, TResponse> method, string? host, CallOptions options, TRequest request)
```

Comment:
Why are we allowing a null host in all these APIs?

Reply:
Generated code passes in null - https://github.com/grpc/grpc/blob/6d287405735da0282f92fb07ab7c56aeb80abc0c/src/csharp/Grpc.Examples/MathGrpc.cs#L224

There is also some Grpc.Core.Api code that passes down this code path, but I don't recall exactly where it is to link to it.

Reply:
`host == null` means "use the default host / authority". You can set "host" explicitly if you want to use multiple virtual hosts on a single host.
See "Authority" in https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md

So setting `host=null` is actually the most common use case.